documentation/release-process.md

```diff
@@ -7,7 +7,7 @@
 1. In `/eng/Version.props`, ensure that `<BlobGroupBuildQuality>` is set appropriately. See the documentation next to this setting for the appropriate values. In release branches, its value should be `release`. This setting, in combination with the version settings, determine for which 'channel' the aks.ms links are created.
```

Comment:
I can't leave the comment on the exact line, but for Step 1 - is that advice accurate when doing a patch release? My understanding is that we wouldn't want to take all of the changes in `main`, but I also haven't done a patch release yet.

Reply:
For patch releases, there are no merges. Fixes are made directly in the release branch.